site/content/docs/5.1/components/dropdowns.md

```diff
@@ -203,7 +203,7 @@ Opt into darker dropdowns to match a dark navbar or custom style by adding `.dro
       <li><a class="dropdown-item" href="#">Another action</a></li>
       <li><a class="dropdown-item" href="#">Something else here</a></li>
       <li><hr class="dropdown-divider"></li>
-      <li><a class="dropdown-item" href="#">Separated link</a></li>
+      <li><a class="dropdown-item disabled" href="#">Separated link</a></li>
```

Comment:
At the end don't forget to remove this modification (and the one line 221). Or rather make the element really `disabled` (by using the attribute) but before the `<hr>` so that we can see that the focus jump over this item.